webrtc-stats.html

```diff
@@ -268,6 +268,25 @@
           PeerConnection.
         </p>
       </section>
+      <section>
+        <h3>
+          Guidelines for getStats() results caching/throttling
+        </h3>
+        <p>
+          A call to getStats() touches many components of WebRTC and may take significant time to
+          execute. The implementation may or may not utilize caching or throttling of getStats()
+          calls for performance benefits, however any implementation must adhere to the following:
+        </p>
+        <p>
+          When the state of the PeerConnection visibly changes as a result of an API call, a
+          promise resolving or an event firing, subsequent new getStats() calls MUST return up-to-date
+          dictionaries for the affected objects. For example, if a track is added with addTrack()
+          subsequent getStats() calls MUST resolve with a corresponding RTCMediaStreamTrackStats
+          object. If you call setRemoteDescription() removing a remote track, upon the promise
+          resolving or an associated event (stream's onremovetrack or track's onmute) firing,
+          calling getStats() MUST resolve with an up-to-date RTCMediaStreamTrackStats object.
```

Comment:
Should be lower-case MUST throughout since this section is non-normative.

Reply:
Done